app/views/teachers/edit.html.erb

```diff
@@ -1,4 +1,11 @@
 <h1>Attinderance | Edit Teacher</h1>
+<p>This page will allow the user/teacher to edit or update the data for their own profile. Right now that is only one string field for name. </p>
+<p>Please enter the name again: </p>
+<br>
+<p><%= render 'form'%>
+   <%= link_to 'Back', students_path%>
```

Comment:
This is something I've notice is that you don't have any space between the ruby and ending ERB tag. It's functional but harder to read this way. If you decide to change this, don't worry about fixing all of these at once. Maybe just fix them as you find them and do this going forward.